src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -172,20 +175,48 @@ <T> Stream<T> getDependencies(
       return function.apply(originalConfiguration.getResolvedConfiguration());
     }
     catch (ResolveException e) {
-      Configuration copyConfiguration = createCopyConfiguration(project);
+      Configuration copyConfiguration = copyDependencies(project, originalConfiguration, false);
 
-      originalConfiguration.getAllDependencies().all(dependency -> {
-        if (dependency instanceof ProjectDependency) {
-          Project dependencyProject = ((ProjectDependency) dependency).getDependencyProject();
-          project.evaluationDependsOn(dependencyProject.getPath());
-        }
-        else {
-          copyConfiguration.getDependencies().add(dependency);
+      try {
+        return function.apply(copyConfiguration.getResolvedConfiguration());
+      }
+      catch (Exception lastResortException) {
+        copyConfiguration = copyDependencies(project, originalConfiguration, true);
+        return function.apply(copyConfiguration.getResolvedConfiguration());
+      }
+    }
+  }
+
+  private Configuration copyDependencies(
+      Project project,
+      Configuration originalConfiguration,
+      boolean skipUnresolvableDependencies)
+  {
+    Configuration copyConfiguration = createCopyConfiguration(project);
+
+    originalConfiguration.getAllDependencies().all(dependency -> {
+      if (dependency instanceof ProjectDependency) {
+        Project dependencyProject = ((ProjectDependency) dependency).getDependencyProject();
+        project.evaluationDependsOn(dependencyProject.getPath());
+      }
+      else {
+        copyConfiguration.getDependencies().add(dependency);
+
+        if (skipUnresolvableDependencies) {
+          try {
+            originalConfiguration.files(dependency);
+          }
+          catch (Exception e) {
+            log.warn("Unable to process the dependency " + dependency.getGroup() + ":" + dependency.getName() + ":"
+                + dependency.getVersion() + " in project " + project.getName() + " and configuration "
+                + originalConfiguration.getName());
```

Comment:
Suggestion. Maybe better to use placeholders rather then string concatenate e.g.
```            log.warn("Unable to process the dependency {}:{}:{} in project {} and configuration {}",
                dependency.getGroup(), dependency.getName(), dependency.getVersion(), dependency.getName(), originalConfiguration.getName());
```

Reply:
> Suggestion. Maybe better to use placeholders rather then string concatenate e.g. `log.warn("Unable to process the dependency {}:{}:{} in project {} and configuration {}", dependency.getGroup(), dependency.getName(), dependency.getVersion(), dependency.getName(), originalConfiguration.getName()); `

Done: 67073e7b27d33ade3f14e05627408a38e954715d